draft-ietf-teep-architecture.md

```diff
@@ -638,9 +650,9 @@ in a device authenticates a TAM. The
 provisioning of Trust Anchors to a device may be different from
 one use case to the other. A Device Administrator may want to
 have the capability to control what TAs are allowed.
-A device manufacturer enables verification of the TAM providers and TA binary signers; 
+A device manufacturer enables verification by one or more TAMs and by TA developers; 
```

Comment:
I think the change from "TA binary signers" to "TA developers" should not be done ("TA signers" would be fine).